WordPress/Classes/Networking/ServiceRemoteREST.h

```diff
@@ -1,5 +1,12 @@
 #import <Foundation/Foundation.h>
 
+typedef enum
+{
+    ServiceRemoteRESTApiVersion_1_1 = 1001,
+    ServiceRemoteRESTApiVersion_1_2 = 1002,
+    ServiceRemoteRESTApiVersionDefault = ServiceRemoteRESTApiVersion_1_1,
+} ServiceRemoteRESTApiVersion;
```

Comment:
Why not use the constant strings directly instead of having to `switch` an enum to get them?